src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
This file and `src/sphinxcontrib-disqus` do not belong in the repo I don't think?